com.unity.render-pipelines.universal/Shaders/Unlit.shader

```diff
@@ -55,8 +59,8 @@ Shader "Universal Render Pipeline/Unlit"
             #pragma multi_compile_fragment _ _DBUFFER_MRT1 _DBUFFER_MRT2 _DBUFFER_MRT3
             #pragma multi_compile _ DEBUG_DISPLAY
 
-            #pragma vertex UniversalVertexUnlit
-            #pragma fragment UniversalFragmentUnlit
+            #pragma vertex UnlitPassVertex
+            #pragma fragment UnlitPassFragment
```

Comment:
This had to be renamed as there's a function in shader library with same name and that caused compiler error in DXC.